scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -916,3 +917,129 @@ object BigQueryTyped {
     }
   }
 }
+
+// SELECT
+
+object MagnolifyBigQuerySelect {
+  type ReadParam = BigQueryTypedSelect.ReadParam
+  val ReadParam = BigQueryTypedSelect.ReadParam
+}
+
+final case class MagnolifyBigQuerySelect[T: TableRowType: Coder](
```

Comment:
```suggestion
final case class BigQueryMagnolifyTypedSelectIO[T: TableRowType: Coder](
```
In BQ we've not been consistantly using the IO suffix, so this can prob be omitted and changed to `BigQueryMagnolifyTypedSelect`